src/screens/simulation/PublicSimulation.js

```diff
@@ -328,21 +354,25 @@ const Legend = ({
 };
 
 const initialValues = {
-    initial_start_date: new Date('2020-01-09'),
+    initial_start_date: new Date('2020-01-06'),
     initial_r0: 3.4,
     lockdown_start_date: new Date('2020-03-17'),
     lockdown_r0: 0.5,
+    lockdown_enabled: true,
     deconfinement_start_date: new Date('2020-05-11'),
     deconfinement_r0: 1.1,
+    deconfinement_enabled: true,
 };
 
 const getTimeframesFromValues = ({
     initial_start_date,
     initial_r0,
     lockdown_start_date,
     lockdown_r0,
+    lockdown_enabled,
     deconfinement_start_date,
     deconfinement_r0,
+    deconfinement_enabled,
```

Comment:
What the english word for deconfinement? We should use english or not english at all